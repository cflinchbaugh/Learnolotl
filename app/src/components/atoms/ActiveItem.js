import React from 'react';
import styled from 'styled-components';

const StyleWrapper = styled.div`
    color: ${props => props.theme.dark};
    font-size: 3em;
    text-shadow: 0px 6px 3px rgba(0,0,0,0.3),
         0px 8px 13px rgba(0,0,0,0.1),
         0px 18px 23px rgba(0,0,0,0.1);
`;


function ActiveItem (props) {
    return (
        <StyleWrapper>
            { props.value }
        </StyleWrapper>
    )
}

export default ActiveItem