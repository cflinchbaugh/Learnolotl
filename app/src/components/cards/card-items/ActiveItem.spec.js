import React from 'react';
import {shallow} from 'enzyme';
import ActiveItem from './ActiveItem';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('ActiveItem', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const tree = renderer.render(
            <ActiveItem />
        );

        expect(tree).toMatchSnapshot();
    });

    it('renders passed in value and matches snapshot', () => {
        const renderer = new ShallowRenderer();
        const activeItemData = {
            value: "passedInValue"
        }

        const tree = renderer.render(
            <ActiveItem {...activeItemData}/>
        );

        expect(tree).toMatchSnapshot();
    });
});