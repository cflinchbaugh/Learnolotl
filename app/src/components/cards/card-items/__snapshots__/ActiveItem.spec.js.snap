// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActiveItem renders and matches snapshot 1`] = `<styled.div />`;

exports[`ActiveItem renders passed in value and matches snapshot 1`] = `
<styled.div>
  passedInValue
</styled.div>
`;
